application/cloud/cloudfoundry.py

```diff
@@ -13,3 +13,10 @@ def detected(self):
     @property
     def db(self):
         return self._cf_env.get_service(name='ras-ci-db')
+
+    @property
+    def queue(self):
+        try:
+            return self._cf_env.get_service(name='ras-rabbitmq')
+        except StopIteration:
```

Comment:
Looks like the stop iteration is handled in get_service, the next default is set to None.